src/bytes.rs

```diff
@@ -114,15 +114,6 @@ pub struct CharIterator {
     pub has_read_eof: bool, // True when we just read an EOF
 }
 
-pub enum SeekMode {
```

Comment:
I've removed the possibility to seek a stream. For now, that is ok enough, and it saves a lot of time processing newlines and column end calculations.